application/controllers/campaign.php

```diff
@@ -917,7 +917,7 @@ public function status($id = null, $component = null, $selected_milestone = null
                         $expected_datajson_url = urldecode($url_override);
                     }
 
-                    $expected_datajson_url = $this->campaign->filter_remote_url($expected_datajson_url);
+                    $expected_datajson_url = filter_remote_url($expected_datajson_url);
```

Comment:
Helpers get included everywhere? No namespace or an anything like that?

Reply:
I'm pretty sure this is what does it, but honestly not sure.

https://github.com/GSA/project-open-data-dashboard/blob/f0c63cc40ec8233428bdaa9fbf790b57011b5897/application/controllers/campaign.php#L10

Reply:
These docs seem to back this up https://www.codeigniter.com/userguide3/general/helpers.html